apps/web/src/app/(dashboard)/admin/documents/data-table.tsx

```diff
@@ -58,14 +60,24 @@ export const DocumentsDataTable = ({ results }: DocumentsDataTableProps) => {
             header: 'Owner',
             accessorKey: 'owner',
             cell: ({ row }) => {
+              const avatarFallback = row.original.User.name
+                ? recipientInitials(row.original.User.name)
+                : row.original.User.email.slice(0, 1).toUpperCase();
               return (
-                <Link href={`/admin/users/${row.original.User.id}`}>
-                  <Avatar className="dark:border-border h-12 w-12 border-2 border-solid border-white">
-                    <AvatarFallback className="text-gray-400">
-                      <span className="text-xs">{row.original.User.name}</span>
-                    </AvatarFallback>
-                  </Avatar>
-                </Link>
+                <Tooltip>
+                  <TooltipTrigger>
+                    <Link href={`/admin/users/${row.original.User.id}`}>
+                      <Avatar className="dark:border-border h-12 w-12 border-2 border-solid border-white">
+                        <AvatarFallback className="text-gray-400">
+                          <span className="text-xs">{avatarFallback}</span>
+                        </AvatarFallback>
+                      </Avatar>
+                    </Link>
+                  </TooltipTrigger>
+                  <TooltipContent className="max-w-xs">
+                    {row.original.User.name ?? row.original.User.email}
+                  </TooltipContent>
+                </Tooltip>
```

Comment:
Can you make it look something like this?

![image](https://github.com/documenso/documenso/assets/20962767/bfb90121-bd14-4522-bb0b-a7658279f82c)

Hover on the avatars on the `/documents` to see a similar design